forge/__init__.py

```diff
@@ -1,22 +1,28 @@
+""" Forge """
 #! /usr/bin/env python3
-
 import sys
-from pluginbase import PluginBase
+import os
+import configparser
 from pathlib import Path
+from pluginbase import PluginBase
 from tabulate import tabulate
 
-plugin_base = PluginBase(package='plugins')
 
+PLUGIN_BASE = PluginBase(package='plugins')
 
-class Application(object):
 
+class Application:
+    """ Application Class """
     def __init__(self, name):
         self.name = name
 
         self.registry = {}
 
-        self.plugin_source = plugin_base.make_plugin_source(
-            searchpath=['/usr/local/etc/plugins'],
+        self.plugins = []
```

Comment:
All paths should be relative and using the OS library...

import os

        dirname = os.path.dirname(__file__)
        filename = os.path.join(dirname, relative_path_file)

This won't work on Windows at all.  If you don't want a relative path, then the dir must be configurable someplace.

Reply:
Gotcha, Ill go with making it configurable for now. I definitely want to support windows installations. Will make a ticket for supporting that.